src/index.html

```diff
@@ -22,6 +22,362 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="/src/images/logo.png"
+          alt="Moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="#Laptops $ Computer"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="Gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Tablet"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div
+          data-qa="card"
+          class="product-card"
+        >
+          <img
+            class="product-card__img"
+            src="/src/images/imac.jpeg"
```

Comment:
The src attribute of img tag is not correct. It should be relative path like './images/imac.jpeg' instead of absolute path like '/src/images/imac.jpeg'. This issue is repeated in the entire file.